src/main/java/org/isf/menu/gui/UserGroupBrowsing.java

```diff
@@ -149,17 +134,43 @@ public UserGroupBrowsing() {
 				try {
 					if (answer == JOptionPane.YES_OPTION) {
 						userBrowsingManager.deleteGroup(userGroup);
-						pGroup.remove(table.getSelectedRow());
 						model.fireTableDataChanged();
 						table.updateUI();
 					}
```

Comment:
why you removed this line?

Reply:
Oops ! I've removed it in a wrong place.  I wanted to keep the item in the list after soft deletion.